### DISCUSSION TITLE
Welcome to this.jobs Discussions!

### DISCUSSION THREAD

**Discussion by flash29** [flash29/this.jobs#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VishnuJammula** [comment]
@flash29 @vangmayi98 what type of attachments are we looking at while post creation?

**Reply by flash29** [reply]
write now let us limit it only to images. @VishnuJammula

**Comment by VishnuJammula** [comment]
@flash29 @vangmayi98 delete sprint 1 branches before creating sprint 2 branches

**Comment by VishnuJammula** [comment]
@flash29 @vangmayi98 Should i prefetch the job applicants for a job posting while retrieving all the job posts or should it be otherwise?

**Reply by flash29** [reply]
If we prefetch it we would unnecessarily be retrieving a lot of data we don’t want like for jobs posting in which the user is not interested to see the applicants.
So return all the jobs posted by the employer along with job ids and then we should be able to retrieve the job applicants using the job id.

**Comment by SnehaParuchuri** [comment]
@vangmayi98 @flash29 are we planning to display (date) when the two users have connected with each other in UI?

**Comment by VishnuJammula** [comment]
@flash29 @SnehaParuchuri @vangmayi98 The people search API takes query parameter with key "search" like **<URL>?search=hello** The api search people upon their email, name and bio and returns {user_id, user_email, user_name, display picture, bio} fields. Let me know if you need any change.

**Comment by VishnuJammula** [comment]
@flash29 @SnehaParuchuri @vangmayi98 Id of user creating the feed post is mandatory from now. Please incorporate this change in the frontend.
